pyproject.toml

```diff
@@ -62,6 +62,7 @@ format-nongpl = [
   "jsonpointer>1.13",
   "rfc3339-validator",
   "rfc3986-validator>0.1.0",
+  "rfc3987-syntax",
```

Comment:
TODO: add proper minimum version
depends on https://github.com/willynilly/rfc3987-syntax/pull/2 and the follow-up release

Reply:
done